clients/drcachesim/tracer/raw2trace.h

```diff
@@ -797,12 +797,23 @@ template <typename T> class trace_converter_t {
                     return err;
                 buf += trace_metadata_writer_t::write_marker(
                     buf, (trace_marker_type_t)in_entry->extended.valueB, marker_val);
-                if (in_entry->extended.valueB == TRACE_MARKER_TYPE_KERNEL_EVENT) {
-                    impl()->log(4, "Signal/exception between bbs\n");
+                if (in_entry->extended.valueB == TRACE_MARKER_TYPE_FUNC_ARG ||
```

Comment:
It seems like any possible intra-block marker needs to be appended by handle_kernel_interrupt_and_markers() so this should include all markers.  Please augment the comments in handle_kernel_interrupt_and_markers() to make this clear as well ("only at block boundaries" comment plus up top?).

Reply:
But it should not include headers: cpu markers.

Reply:
Done. Let me know if that's what you had in mind.